[2023-12-11T22:36:44.560+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_dag_v02.insert_into_postgres_table scheduled__2023-12-01T02:00:00+00:00 [queued]>
[2023-12-11T22:36:44.910+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_dag_v02.insert_into_postgres_table scheduled__2023-12-01T02:00:00+00:00 [queued]>
[2023-12-11T22:36:44.933+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-12-11T22:36:45.246+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_postgres_table> on 2023-12-01 02:00:00+00:00
[2023-12-11T22:36:46.160+0000] {standard_task_runner.py:57} INFO - Started process 10510 to run task
[2023-12-11T22:36:46.282+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'postgres_dag_v02', 'insert_into_postgres_table', 'scheduled__2023-12-01T02:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/postgres_dag.py', '--cfg-path', '/tmp/tmpx9unj_95']
[2023-12-11T22:36:47.653+0000] {standard_task_runner.py:85} INFO - Job 278: Subtask insert_into_postgres_table
[2023-12-11T22:36:51.109+0000] {task_command.py:416} INFO - Running <TaskInstance: postgres_dag_v02.insert_into_postgres_table scheduled__2023-12-01T02:00:00+00:00 [running]> on host d5f251ea7e03
[2023-12-11T22:36:51.913+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sajal' AIRFLOW_CTX_DAG_ID='postgres_dag_v02' AIRFLOW_CTX_TASK_ID='insert_into_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T02:00:00+00:00'
[2023-12-11T22:36:52.048+0000] {sql.py:274} INFO - Executing: 
        insert into dag_runs(dt, dag_id) values ( '2023-12-01', 'postgres_dag_v02')
    
[2023-12-11T22:36:52.132+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-11T22:36:52.427+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-11T22:36:53.142+0000] {sql.py:418} INFO - Running statement: 
        insert into dag_runs(dt, dag_id) values ( '2023-12-01', 'postgres_dag_v02')
    , parameters: None
[2023-12-11T22:36:53.220+0000] {sql.py:427} INFO - Rows affected: 1
[2023-12-11T22:36:53.459+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=postgres_dag_v02, task_id=insert_into_postgres_table, execution_date=20231201T020000, start_date=20231211T223644, end_date=20231211T223653
[2023-12-11T22:36:56.046+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-11T22:37:06.310+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
